make-nagios-commands-cfg.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+import os
+
+from jinja2 import Environment, FileSystemLoader, Template
+
+smtp_server = os.getenv('SMTP_SERVER', 'localhost')
```

Comment:
I get your point,  I don't like the magic fallback values either, but this is extra work that was not in the scope of what we planned here. I think we should add a new ticket in the backlog and discuss as a team how best to deal with this and look to resolve it in a separate PR.